### DISCUSSION TITLE
Release v0.3.0-beta2

### DISCUSSION THREAD

**Discussion by f04ever** [f04ever/noso-2m#18] [Announcements]
- IP binding support: If there are more than one valid IP addresses on your machine, you can launch two noso-2m instances, bind to those two different IP addresses. 
- PoPW compatible.
- Exceptions/errors on android versions have been determined and fixed.

Check noso-2m-SAMPLE.cfg, noso-2m-SAMPLE-IPv4POOL.cfg, and ./noso-2m --help for more information.

*** This is the beta version, built without TEXTUI as it still need to re-design the UI.

Welcome and appreciate any comments! (I may not reply your comments, but i always listen to your ideas)

Happy mining!

<hr /><em>This discussion was created from the release <a href='https://github.com/f04ever/noso-2m/releases/tag/v0.3.0-beta2'>Release v0.3.0-beta2</a>.</em>

**Comment by ahmedkoko5253** [comment]
I want to link the wallet address in the mining application, where can I get it from?

**Reply by f04ever** [reply]
At launch you can find your wallet address like in the picture
<img width="757" alt="Screenshot 2023-01-25 at 10 15 48" src="https://user-images.githubusercontent.com/86355159/214525051-da3d8c30-37df-47cf-b991-563529c43beb.png">

Best!
